src/pages/Configuration.tsx

```diff
@@ -347,33 +347,49 @@ export default function Configuration({ configuration }: ConfigurationParams) {
           also available on the <StyledLink to={'/sgp'}>SGP Management</StyledLink> page as "Work Requester"
         </p>
         <EntityManager
-          initialEntities={convertToBasicReleaseRecipient(configuration.releaseRecipients)}
+          initialEntities={configuration.releaseRecipients.map((rr) => ({
+            ...rr,
+            fullName: rr.fullName || ''
+          }))}
           displayKeyColumnName={'username'}
           valueColumnName={'enabled'}
           onChangeValue={(entity, value) => {
             const enabled = typeof value === 'boolean' ? value : false;
             return stanCore
               .SetReleaseRecipientEnabled({
                 enabled,
-                username: entity.username
+                username: String(entity.username)
               })
-              .then((res) => convertToBasicReleaseRecipient(res.setReleaseRecipientEnabled)[0]);
-          }}
-          onCreate={(username, userFullName) => {
-            return stanCore
-              .AddReleaseRecipient({ username, userFullName })
-              .then((res) => convertToBasicReleaseRecipient(res.addReleaseRecipient)[0]);
+              .then((res) => {
+                return {
+                  ...res.setReleaseRecipientEnabled,
+                  fullName: res.setReleaseRecipientEnabled.fullName || ''
+                };
+              });
+          }}
+          onCreate={(username, fullName) => {
+            return stanCore.AddReleaseRecipient({ username, fullName }).then((res) => {
+              return {
+                ...res.addReleaseRecipient,
+                fullName: res.addReleaseRecipient.fullName || ''
+              };
+            });
           }}
           valueFieldComponentInfo={{
             type: 'CHECKBOX'
           }}
           extraDisplayColumnName={{
-            label: 'User Full Name',
-            value: 'userFullName',
-            onChange: (username, userFullName) => {
-              return stanCore
-                .UpdateReleaseRecipientFullName({ username, userFullName })
-                .then((res) => convertToBasicReleaseRecipient(res.updateReleaseRecipientFullName)[0]);
+            label: 'Full Name',
+            value: 'fullName',
+            extraFieldPlaceholder: 'Enter User Full Name',
```

Comment:
Probably you can change this as well to  'Enter full name' to match column header